soc/arm/nxp_lpc/Kconfig

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2017, NXP
+# Copyright (c) 2019, NXP
```

Comment:
Don't remove existing copyrights. I meant in https://github.com/zephyrproject-rtos/zephyr/pull/15647#discussion_r279887544 to update the copyright in new files.

Reply:
@MaureenHelm made the changes you required